src/match/controllers/create_match_controller.py

```diff
@@ -16,31 +16,39 @@ def process(self):
         match_ids = self._data.get("ids")
         if match_ids is None:
             error_msg = "POST body is misformatted"
-            return False, error_msg if self._return_status else failure_response(
-                error_msg, status.HTTP_400_BAD_REQUEST
+            return (
+                (False, error_msg)
+                if self._return_status
+                else failure_response(error_msg, status.HTTP_400_BAD_REQUEST)
             )
         # Always set user_1 to the lower user id
         user_1 = User.objects.filter(id=min(match_ids))
         if not user_1:
             error_msg = f"User with id {min(match_ids)} does not exist"
-            return False, error_msg if self._return_status else failure_response(
-                error_msg
+            return (
+                (False, error_msg)
+                if self._return_status
+                else failure_response(error_msg)
             )
         # Always set user_2 to the higher user id
         user_2 = User.objects.filter(id=max(match_ids))
```

Comment:
doesnt this... give you a list? how are you even accessing user_2 rn..

`User.objects.filter(id=max(match_ids)).exists()` is what you're looking for? it's a `bool`

Reply:
Actually scroll down to my later comment, @alanna-zhou suggested a cleaner, clearer way of doing this.